File: TextShadow.java
package ja.burhanrashid52.photoeditor;

public class TextShadow {
    float radius;
    float dx;
    float dy;
    int color;

    public TextShadow(float radius, float dx, float dy, int color) {
        this.radius = radius;
        this.dx = dx;
        this.dy = dy;
        this.color = color;
    }

    public float getRadius() {
        return radius;
    }

    public void setRadius(float radius) {
        this.radius = radius;
    }

    public float getDx() {
        return dx;
    }

    public void setDx(float dx) {
        this.dx = dx;
    }

    public float getDy() {
        return dy;
    }

    public void setDy(float dy) {
        this.dy = dy;
    }

    public int getColor() {
        return color;
    }

    public void setColor(int color) {
        this.color = color;
    }
}

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextShadow.kt
package ja.burhanrashid52.photoeditor

class TextShadow(private var radius: Float, private var dx: Float, private var dy: Float, private var color: Int) {
    fun getRadius(): Float {
        return radius
    }

    fun setRadius(radius: Float) {
        this.radius = radius
    }

    fun getDx(): Float {
        return dx
    }

    fun setDx(dx: Float) {
        this.dx = dx
    }

    fun getDy(): Float {
        return dy
    }

    fun setDy(dy: Float) {
        this.dy = dy
    }

    fun getColor(): Int {
        return color
    }

    fun setColor(color: Int) {
        this.color = color
    }
}